### ISSUE TITLE
Email call to pre-register sites for checking

### ISSUE THREAD

**Issue by JimKillock** [openrightsgroup/cmp-issues#22]
We could ask people to submit sites for the checking queue, so they're in the database prior to launch. This would act as a teaser for the launch to build interest for when it happens.

**Comment by graphiclunarkid** [comment]
Also available by Twitter thanks to @NetworksAreMadeOfString: OrgBlockBot

**Comment by JimKillock** [comment]
OK, so technically people can do this already?!

**Comment by JimKillock** [comment]
I guess, of course they can …

**Comment by graphiclunarkid** [comment]
_TL;DR: I think keeping this to Twitter rather than email might be a good idea._

Right now I think URLs can be submitted to the queue via any of these methods:
- https://twitter.com/orgblockbot
- Anyone running https://github.com/openrightsgroup/Android-Probe
- http://stage.blocked.org.uk/
- http://new.blocked.org.uk
- https://www.blocked.org.uk/

To which of these are we going to direct people in this email? What if people reply to the email with "please add my email domain to your checking list"?

Ideally any emails from the project would come from (blocked at org dot org) however that's not set up yet (see #2). Who is going to handle all these emails? (We still don't have a CRM system running for this - an email thread is ongoing...)

**Comment by JimKillock** [comment]
We'd direct people to https://www.blocked.org.uk/ - purely as a teaser. The emails would come from ORG as normal. 

We shouldn't have to handle anything, really, as it would just mean people enter info into the database.

I suppose there is the problem of people opting into further contact, as that's not managed at all as you say. The data would be in FormSave however for export when we're ready.
